reconcile/test/test_ocm_machine_pools.py

```diff
@@ -32,7 +32,7 @@
 from reconcile.utils.ocm import OCM
 
 
-class TestPool(AbstractPool):
+class TstPool(AbstractPool):
```

Comment:
Since this is only used in test as stubs, a more suitable name would be `PoolStub`

Reply:
`...Stub` is much better. Changed in the last commit. Thanks